### ISSUE TITLE
send: "current access count" not updating in web vault

### ISSUE THREAD

**Issue by fbartels** [dani-garcia/bitwarden_rs#1557]
### Subject of the issue

The send feature allow to expire items once it has been requested a given amount of times. This in itself also word (e.g. saying a link should have a "maximum access count" of two, no longer works when visiting it for the third time. However the gui also has a field "current access count" which looks like it should show how often an item has been requested already, but this seems to be stuck a "0".

### Deployment environment

* bitwarden_rs version: 1.20.0
* Install method: using bitwardenrs/server:1.20.0
* Clients used: bundled web vailt
* Reverse proxy and version: Apache
* MySQL/MariaDB or PostgreSQL version: sqlite
* Other relevant details: none

### Steps to reproduce

- go to `https://domain/#/sends` and create a new send
- expand "options" and set "maximum access count to e.g. "2"
- click save and visit the link

### Expected behaviour

- after visiting the link the "current access" should have increased from "0" to "1" 

### Actual behaviour

- the counter stays at "0"

**Comment by fbartels** [comment]
After opening this issue I went to the Chrome extension to see how this is displayed there and there the counter was successfully increased. Curiously after opening the element again in the web vault also there it now showed the correct amount of requests. I did make sure to reload the web vault before, but its counter stayed at zero.

Screenshot showing the web vault and the browser extension side by side:
![image](https://user-images.githubusercontent.com/1257835/112823672-de558d00-9089-11eb-9130-e8c91bf08a1f.png)
